src/workflows-app/SubmitWorkflow.js

```diff
@@ -0,0 +1,126 @@
+import { Fragment, useCallback, useState } from 'react';
+import { div, h, h2 } from 'react-hyperscript-helpers';
+import { ButtonOutline, Clickable } from 'src/components/common';
+import { centeredSpinner, icon } from 'src/components/icons';
+import { Ajax } from 'src/libs/ajax';
+import colors from 'src/libs/colors';
+import { isFindWorkflowEnabled } from 'src/libs/config';
+import * as Nav from 'src/libs/nav';
+import { notify } from 'src/libs/notifications';
+import { useCancellation, useOnMount } from 'src/libs/react-utils';
+import * as Style from 'src/libs/style';
+import { withBusyState } from 'src/libs/utils';
+import FindWorkflowModal from 'src/workflows-app/components/FindWorkflowModal';
+import { SavedWorkflows } from 'src/workflows-app/components/SavedWorkflows';
+import { loadAppUrls } from 'src/workflows-app/components/submission-common';
+import { wrapWorkflowsPage } from 'src/workflows-app/WorkflowsContainer';
+
+const styles = {
+  // Card's position: relative and the outer/inner styles are a little hack to fake nested links
+  card: {
+    ...Style.elements.card.container,
+    position: 'absolute',
+  },
+  shortCard: {
+    width: 300,
+    height: 125,
+    margin: '0 1rem 2rem 0',
+  },
+};
+
+export const SubmitWorkflow = wrapWorkflowsPage({ name: 'SubmitWorkflow' })(
+  (
+    {
+      name,
+      namespace,
+      workspace: {
+        workspace: { workspaceId },
+      },
+    },
+    _ref
+  ) => {
+    const [methodsData, setMethodsData] = useState();
+    const [loading, setLoading] = useState(false);
+    const [viewFindWorkflowModal, setViewFindWorkflowModal] = useState(false);
+
+    const signal = useCancellation();
+
+    const refreshAppUrls = useCallback(async () => {
+      const {
+        wds: { state: wdsUrlRoot },
+        cbas: { state: cbasUrlRoot },
+      } = await loadAppUrls(workspaceId);
+
+      return { wdsUrlRoot, cbasUrlRoot };
+    }, [workspaceId]);
+
+    const loadRunsData = useCallback(async () => {
+      const { cbasUrlRoot } = await refreshAppUrls();
+      try {
+        const runs = await Ajax(signal).Cbas.methods.getWithoutVersions(cbasUrlRoot);
+        setMethodsData(runs.methods);
+      } catch (error) {
+        notify('error', 'Error loading saved workflows', { detail: error instanceof Response ? await error.text() : error });
+      }
+    }, [refreshAppUrls, signal]);
+
+    const refresh = withBusyState(setLoading, async () => {
+      await loadRunsData();
+    });
+
+    useOnMount(() => {
+      refresh();
+    });
+
+    return loading
+      ? centeredSpinner()
+      : div([
+          div({ style: { margin: '4rem' } }, [
+            div({ style: { display: 'flex', marginTop: '1rem', justifyContent: 'space-between' } }, [
+              h2(['Submit a workflow']),
```

Comment:
small nit: Maybe we can reduce some space between the blue grey bar and "Submit workflow and submission history" div here.

![Screenshot 2023-07-05 at 6 02 09 PM](https://github.com/DataBiosphere/terra-ui/assets/16748522/6b1f88c4-3f2f-4ac5-97ee-2591a3b050b3)